staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/admission.go

```diff
@@ -248,7 +248,6 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 				if ignoreClientCallFailures {
 					glog.Warningf("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
 					utilruntime.HandleError(callErr)
-					// Since we are failing open to begin with, we do not send an error down the channel
```

Comment:
Are we failing open? If so, leave a comment, if not, perhaps we should send an error down the channel?

Reply:
fail open v.s. fail closed is captured by `ignoreClientCallFailures`.